decidim-core/lib/decidim/filter_form_builder.rb

```diff
@@ -5,93 +5,60 @@
 module Decidim
   # This custom FormBuilder is used to create resource filter forms
   class FilterFormBuilder < FormBuilder
-    # Wrap the radio buttons collection in a custom fieldset.
-    # It also renders the inputs inside its labels.
-    def collection_radio_buttons(method, collection, value_method, label_method, options = {}, html_options = {})
-      fieldset_wrapper(options[:legend_title], "#{method}_collection_radio_buttons_filter") do
-        super(method, collection, value_method, label_method, options, html_options) do |builder|
-          if block_given?
-            yield builder
-          else
-            builder.label { builder.radio_button + builder.text }
-          end
-        end
-      end
-    end
-
-    # Wrap the check_boxes collection in a custom fieldset.
-    # It also renders the inputs inside its labels.
-    def collection_check_boxes(method, collection, value_method, label_method, options = {}, html_options = {})
-      fieldset_wrapper(options[:legend_title], "#{method}_collection_check_boxes_filter") do
-        super(method, collection, value_method, label_method, options, html_options) do |builder|
-          if block_given?
-            yield builder
-          else
-            builder.label { builder.check_box + builder.text }
-          end
-        end
-      end
-    end
+    # This method is used to generate a section of options in a filter block
+    # @param method [Symbol] - The method associated to the filter object of the form builder.
+    # @param collection [Array] - The collection used to display the options. It can be an
+    #                                                   array of options where each options if represented by an
+    #                                                   array containing a value and a name or a check_boxes_tree
+    #                                                   struct as defined in Decidim::CheckBoxesTreeHelper for more
+    #                                                   complex situations which require nested options.
+    # @param label_scope [String] - The scope used to translate the title of the section.
+    # @param id [String] - The id of the section. It is also used to get the translation of the
+    #                                      section title.
```

Comment:
Seems like the indentation here is wrong. As I mentioned in my other review, this is a mess to work with GitHub Review UI. Can you fix it please?

Reply:
Moved to #11739